src/frontend/Sider/Behandling/Inngangsvilkår/Målgruppe/RegisterYtelser.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import { styled } from 'styled-components';
+
+import { Alert, Detail, Table, VStack } from '@navikt/ds-react';
+import { ABorderDivider } from '@navikt/ds-tokens/dist/tokens';
+
+import ExpansionCard from '../../../../komponenter/ExpansionCard';
+import { registerYtelseTilTekst } from '../../../../typer/registerytelser';
+import { formaterNullableIsoDato, formaterNullableIsoDatoTid } from '../../../../utils/dato';
+import { VilkårperioderGrunnlag } from '../typer/vilkårperiode';
+
+const TabellContainer = styled(Table)`
+    background: white;
+    --ac-table-row-border: ${ABorderDivider};
+    --ac-table-row-hover: none;
+    --ac-table-cell-hover-border: ${ABorderDivider};
+`;
+
+const RegisterYtelser: React.FC<{ grunnlag: VilkårperioderGrunnlag | undefined }> = ({
+    grunnlag,
+}) => {
+    if (grunnlag === undefined) {
```

Comment:
Hva hvis den er null? 👀 (vår typing tilsier kun undefined, men de kan være null og)
```suggestion
    if (!grunnlag) {
```